hf-llm-inference/Dockerfile

```
@@ -0,0 +1,21 @@
+FROM ghcr.io/coreweave/ml-containers/torch:afecfe9-base-cuda12.0.1-torch2.0.0-vision0.15.1
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt update && apt upgrade -y && \
+    apt update && apt install -y python3 python3-pip git curl && \
```

The `torch` base images include Python, and have PyTorch compiled against that particular one. Reinstalling it here may lead to binary compatibility issues.
```suggestion
    apt update && apt install -y git curl && \
```